app/components/certificate/certificate-display.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Flex,
+  Box,
+  HStack,
+  Text,
+  Heading,
+  Tooltip,
+  IconButton,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionIcon,
+  AccordionPanel,
+  useToast,
+} from '@chakra-ui/react';
+import { DownloadIcon, CopyIcon } from '@chakra-ui/icons';
+import type { Certificate } from '@prisma/client';
+
+interface Props {
+  cert: Certificate;
+  priv: Boolean;
+}
+
+export default function CertificateDisplay(props: Props) {
+  const toast = useToast();
+
+  function onCopy() {
+    let key = props.cert.certificate;
+    let val = 'Public Key';
+
+    if (props.priv) {
```

Comment:
Instead of doing this, pass down the `title` to show (e.g., `'Private Key'`) and also the `value` to use (i.e., the specific PEM string to show).  This component shouldn't have to know anything about what it's showing, do that above it.

Reply:
I'm a bit confused as to what you mean, do you mind explaining it one more time?